# --- T2-COPYRIGHT-BEGIN ---
# t2/architecture/avr32/package/*/hotfix-stdlib.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- binutils-2.18.atmel.1.0.1/libiberty/md5.c.vanilla	2025-04-15 18:44:35.413986802 +0100
+++ binutils-2.18.atmel.1.0.1/libiberty/md5.c	2025-04-15 18:44:55.789282544 +0100
@@ -38,6 +38,7 @@
 
 #include "ansidecl.h"
 #include "md5.h"
+#include <string.h>
 
 #ifdef _LIBC
 # include <endian.h>
--- binutils-2.18.atmel.1.0.1/libiberty/regex.c.vanilla	2025-04-15 18:42:43.879865751 +0100
+++ binutils-2.18.atmel.1.0.1/libiberty/regex.c	2025-04-15 18:44:18.580386861 +0100
@@ -41,6 +41,7 @@
 #endif
 
 #include <ansidecl.h>
+#include <stdlib.h>
 
 #ifndef INSIDE_RECURSION
 
@@ -127,8 +128,8 @@
 #  if defined STDC_HEADERS || defined _LIBC
 #   include <stdlib.h>
 #  else
-char *malloc ();
-char *realloc ();
+//char *malloc ();
+//char *realloc ();
 #  endif
 
 /* When used in Emacs's lib-src, we need to get bzero and bcopy somehow.
